domains/environment_domains/front_door.tf

```diff
@@ -2,16 +2,41 @@ data "azurerm_cdn_frontdoor_profile" "main" {
   name                = var.front_door_name
   resource_group_name = var.resource_group_name
 }
-
 resource "azurerm_cdn_frontdoor_endpoint" "main" {
   for_each = toset(var.domains)
 
-  name                     = each.value
+  name                     = "${each.value}-${local.endpoint_zone_name}"
   cdn_frontdoor_profile_id = data.azurerm_cdn_frontdoor_profile.main.id
   tags                     = var.tags
 }
 
-# TODO: Create origin group
-# TODO: Create origin
+resource "azurerm_cdn_frontdoor_origin_group" "main" {
+  name                     = "${var.environment}-og"
+  cdn_frontdoor_profile_id = data.azurerm_cdn_frontdoor_profile.main.id
+  session_affinity_enabled = true
+
+  restore_traffic_time_to_healed_or_new_endpoint_in_minutes = 10
+
+  load_balancing {
```

Comment:
Since we only have 1 origin, can we remove the load_balancing block?

Reply:
This results in following error on terrform run. 
 
 `Error: Insufficient load_balancing blocks. At least 1 "load_balancing" blocks are required.`

Reply:
at least an empty load_balancing block is expected. 

`Error: Insufficient load_balancing blocks. At least 1 "load_balancing" blocks are required.`. 

Adding an empty load_balancing puts defaults as follows:

```
      ~ load_balancing {
          ~ additional_latency_in_milliseconds = 0 -> 50
          ~ sample_size                        = 1 -> 4
          ~ successful_samples_required        = 1 -> 3
        }
```

Reply:
Then maybe simply: `load_balancing {}`